proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -56,6 +95,16 @@ service AdminService {
   }
   // Update breaking change policy for the server.
   rpc UpdateBreakingChangePolicy(UpdateBreakingChangePolicyRequest) returns (UpdateBreakingChangePolicyResponse);
+  // Get uniqueness policy for the server.
+  rpc GetUniquenessPolicy(GetUniquenessPolicyRequest) returns (GetUniquenessPolicyResponse) {
+    option idempotency_level = NO_SIDE_EFFECTS;
+  }
+  // Update uniqueness policy enforcement for the server.
+  rpc UpdateUniquenessPolicy(UpdateUniquenessPolicyRequest) returns (UpdateUniquenessPolicyResponse);
+  // Get state of uniqueness collisions for the server
+  rpc GetServerUniquenessCollisions(GetServerUniquenessCollisionsRequest) returns (GetServerUniquenessCollisionsResponse) {
```

Comment:
I don't love this, but I _think_ this should technically be `ListServerUniquenessCollisions` and have pagination a la https://google.aip.dev/132.